Admin/Test-ExchangePropertyPermissions/Get-ObjectTypeDisplayName.ps1

```diff
@@ -0,0 +1,32 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-ObjectTypeDisplayName {
```

Comment:
Can we see this being used in other scenarios? Might be a good script to move to shared now if it is.